APMrover2/fence.cpp

```diff
@@ -16,13 +16,36 @@ void Rover::fence_check()
 
     // if there is a new breach take action
     if (new_breaches) {
+    	gcs().send_text(MAV_SEVERITY_NOTICE,"Geofence triggered");
+
         // if the user wants some kind of response and motors are armed
-        if (g2.fence.get_action() != AC_FENCE_ACTION_REPORT_ONLY) {
-            // if we are within 100m of the fence, RTL
+        if (g2.fence.get_action() != Failsafe_Action_None) {
+            // if we are within 100m of the fence, HOLD
```

Comment:
This comment is not quite right now.. if within 100m of the fence it will take the action specified by the FENCE_ACTION parameter.